templates/referent/_layout.html.twig

```diff
@@ -56,6 +56,13 @@
                     }
                 ] %}
 
+                {% if app.user.isReferent %}
```

Comment:
this is `referent/_layout.html` template, I think the user cannot be other that referent here